not-without.apl

```diff
@@ -6,5 +6,23 @@
 
 'abc'~'a'
 ⍝ NOTE: if you want to remove *strings* (not chars),
-⍝       you'll need to provide more than one arg, to get a matrix.
-'abc' 'def'~'abc' ''
+⍝       you'll need to enclose the character array you want to remove.
+⍝       In Dyalog, juxtaposing consecutive arrays is called "stranding" and produces a nested vector of vectors.
+⍝ Example session below:
+'      ''abc'' 
```

Comment:
The cases seems not to be valid (or at least, TryAPL doesn't like them).

Reply:
Removed the example session - I'm making Jupyter Notebooks so will be able to include it then. For now there is another pull request